doc/js-api/caching.rst

```diff
@@ -22,6 +43,19 @@ For each subtree, you can set the caching strategy to ``ALL``, ``SEEN``
 * ``FLUSH`` will only cache outgoing changes, and forget them as soon as
   they have been saved to remote successfully.
 
+Synchronizing after caching settings have changed
+-------------------------------------------------
+
+Whenever you have changed the caching settings, you can either wait for
+the next automatic synchronization to happen, or trigger one yourself:
+
+.. code:: javascript
+
+   remoteStorage.startSync().then(function() {
```

Comment:
I can't get this to work, `remoteStorage.startSync()` always returns `undefined` (testing in my RS 1.0 branch of myfavoritedrinks)

Reply:
Just remoteStorage.startSync() works, it doesn't return a promise like sync() used to do

Reply:
That was added after RC1 in 501031b1cb9e91f7ccccb0fcb58cd0c1366a6b95 (is part of RC2)

Reply:
Thanks, I had forgotten to update to RC2. This works as advertised!